TYPO3.Flow/Classes/TYPO3/Flow/Utility/Files.php

```diff
@@ -377,7 +378,11 @@ public static function unlink($pathAndFilename)
         try {
             // if not on Windows, call PHPs own unlink() function
             if (DIRECTORY_SEPARATOR === '/' || is_file($pathAndFilename)) {
```

Comment:
The reason is symlinks, yes. See https://github.com/hlubek/flow-development-collection/commit/04aaa0248744ae3d19f2094e2583fdfde53de268 an/or the `unlink` method in this very file.